src/GAP_pkg.jl

```diff
@@ -77,8 +64,10 @@ function gap_pkg_artifact_dir(pkgname)
 end
 
 function setup_overrides()
-    for pkg in gap_pkgs_with_overrides
-        jll = getproperty(GAP, Symbol("GAP_pkg_$(pkg)_jll"))
+    gap_pkg_jll_names = filter(startswith("GAP_pkg"), String.(names(GAP; imported=true)))
```

Comment:
This filter looks a bit brittle to me, but CI should tell us once we accidentally introduce other variables that match this pattern, right?